viz/charts/linkerd-viz/templates/prometheus.yaml

```diff
@@ -197,15 +198,17 @@ kind: Deployment
 metadata:
   annotations:
     {{ include "partials.annotations.created-by" . }}
+    linkerd.io/inject: enabled
+    config.linkerd.io/proxy-await: "enabled"
```

Comment:
Same as with tap, I think we can maybe remove the await annotation if it's admitted by the injector? (which afaik should add the await postStart hook by default)